sihl/test/web_flash.ml

```diff
@@ -80,44 +93,51 @@ let remove_flash_message _ () =
         Lwt.return res)
       req
   in
-  (* we need to simulate the browser sending back the cookie *)
+  (* Simulating the browser sending the last Set-Cookie *)
   let cookie = Opium.Response.cookie "_flash" res |> Option.get in
   let cookie_value = cookie.Opium.Cookie.value in
   let req = Opium.Request.get "" in
   let req = Opium.Request.add_cookie cookie_value req in
   let* res =
     Rock.Middleware.apply
       (Sihl.Web.Middleware.flash ())
-      (fun _ ->
+      (fun req ->
+        let alert = Sihl.Web.Flash.find_alert req in
+        let notice = Sihl.Web.Flash.find_notice req in
+        Alcotest.(check (option string) "has alert" (Some "foobar") alert);
+        Alcotest.(check (option string) "has no notice" None notice);
         let res = Opium.Response.of_plain_text "" in
-        let res = Sihl.Web.Flash.set_alert None res in
+        let res = Sihl.Web.Flash.set_alert (Some "nextfoo") res in
         let res = Sihl.Web.Flash.set_custom (Some "hello") res in
         Lwt.return res)
       req
   in
+  (* Simulating the browser sending the last Set-Cookie *)
   let cookie = Opium.Response.cookie "_flash" res |> Option.get in
   let cookie_value = cookie.Opium.Cookie.value in
-  let req = Opium.Request.get "" in
-  let req = Opium.Request.add_cookie cookie_value req in
-  let* _ =
+  let req = Opium.Request.get "" |> Opium.Request.add_cookie cookie_value in
+  let* response =
     Rock.Middleware.apply
       (Sihl.Web.Middleware.flash ())
       (fun req ->
         let alert = Sihl.Web.Flash.find_alert req in
         let custom = Sihl.Web.Flash.find_custom req in
-        Alcotest.(check (option string) "has no alert" None alert);
+        Alcotest.(check (option string) "has alert" (Some "nextfoo") alert);
         Alcotest.(check (option string) "has custom" (Some "hello") custom);
         let res = Opium.Response.of_plain_text "" in
         Lwt.return res)
       req
   in
-  let req = Opium.Request.get "" in
+  (* Simulating the browser sending the last Set-Cookie *)
```

Comment:
What was the comment scheme we agreed on? Imperative and passive? If yes, please rewrite to something like:
`Simulate the browser sending the last Set-Cookie`

Reply:
Fixed.